File: CS/LongRunningOperations.Module/BusinessObjects/ObjectForLongRunningOperations.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.Persistent.BaseImpl;
using DevExpress.Persistent.Base;
using DevExpress.Xpo;

namespace LongRunningOperations.Module.BusinessObjects {
    [DefaultClassOptions]
    public class ObjectForLongRunningOperations : BaseObject{
        public ObjectForLongRunningOperations(Session session)
            : base(session) {
        }
        private Object1 obj;
        public Object1 ObjectToProcess {
            get { return obj; }
            set { SetPropertyValue<Object1>("ObejectToProcess", ref obj, value); }
        }
    }
}

File: VB/LongRunningOperations.Module/BusinessObjects/ObjectForLongRunningOperations.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Persistent.Base
Imports DevExpress.Xpo

Namespace LongRunningOperations.Module.BusinessObjects
    <DefaultClassOptions> _
    Public Class ObjectForLongRunningOperations
        Inherits BaseObject

        Public Sub New(ByVal session As Session)
            MyBase.New(session)
        End Sub
        Private obj As Object1
        Public Property ObjectToProcess() As Object1
            Get
                Return obj
            End Get
            Set(ByVal value As Object1)
                SetPropertyValue(Of Object1)("ObejectToProcess", obj, value)
            End Set
        End Property
    End Class
End Namespace
